doc/classes/TextServer.xml

```diff
@@ -1940,5 +1955,14 @@
 		<constant name="STRUCTURED_TEXT_CUSTOM" value="6" enum="StructuredTextParser">
 			User defined structured text BiDi override function.
 		</constant>
+		<constant name="FIXED_SIZE_SCALE_DISABLE" value="0" enum="FixedSizeScaleMode">
+			Bitmap font is not scaled.
+		</constant>
+		<constant name="FIXED_SIZE_SCALE_INTEGER_ONLY" value="1" enum="FixedSizeScaleMode">
+			Bitmap font is scaled to the closest integer multiple of [code]fixed_size[/code].
```

Comment:
Code markup like this is a bit ambiguous as it's not clear which `fixed_size` it's referring to.

Is it `[member FontFile.fixed_size]`, or the `[code]fixed_size[/code] parameter of [method font_set_fixed_size]`, or another more abstract reference?

Reply:
> Is it [member FontFile.fixed_size], or the [code]fixed_size[/code] parameter of [method font_set_fixed_size]

It's reference to both of there. If it's used for the `fixed_size_scale_mode` of the `FontFile` it's member of the same font file. If used in the TextServer API directly, it's parameter of `font_set_fixed_size` call. Since FontFile is a wrapper over TextServer, both can be mixed.

Reply:
Maybe just:
```suggestion
			Bitmap font is scaled to the closest integer multiple of the font's fixed size.
```